.github/workflows/test_windows.yml

```diff
@@ -0,0 +1,30 @@
+name: Run tests on Windows
+
+on: [push]
```

Comment:
do we want to just tigger this for pull requests?

Reply:
Ah I don't know how off the top of my head. `pull_request` is more than that https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

Reply:
Nevermind the default types cover pushes to PRs

Reply:
```suggestion
on: [pull_request]
```